composer.json

```diff
@@ -21,6 +21,7 @@
     "illuminate/support": "^8.73 || ^9 || ^10 || ^11 || ^12",
     "mll-lab/str_putcsv": "^1",
     "nesbot/carbon": "^2.62.1 || ^3",
+    "phpoffice/phpspreadsheet": "^5.1",
```

Comment:
Da sehr speziell würde ich diese Dependency nur in `require-dev` sowie in `suggest` unterbringen wollen.